mmv1/third_party/terraform/utils/compute_instance_helpers.go.erb

```diff
@@ -179,13 +190,15 @@ func flattenNetworkInterfaces(d *schema.ResourceData, config *Config, networkInt
 		region = subnet.Region
 
 		flattened[i] = map[string]interface{}{
-			"network_ip":         iface.NetworkIP,
-			"network":            ConvertSelfLinkToV1(iface.Network),
-			"subnetwork":         ConvertSelfLinkToV1(iface.Subnetwork),
-			"subnetwork_project": subnet.Project,
-			"access_config":      ac,
-			"alias_ip_range":     flattenAliasIpRange(iface.AliasIpRanges),
-			"nic_type":           iface.NicType,
+			"network_ip":          iface.NetworkIP,
```

Comment:
Instance template shares this code, so this is likely to crash it by setting a field that doesn't exist. Does template support configuring this value? We may need to guard it, otherwise.